incubator/kafka/templates/tests/test_topic_create_consume_produce.yaml

```diff
@@ -13,7 +13,13 @@ spec:
     - -c
     - |
       # Create the topic
-      kafka-topics --zookeeper {{ template "zookeeper.url" . }} --topic helm-test-topic-create-consume-produce --create --partitions 1 --replication-factor 1 --if-not-exists && \
+      kafka-topics --zookeeper \
+{{- if not ( index .Values "configurationOverrides" "zookeeper.connect" ) }}
```

Comment:
@josdotso you could wrap this conditional logic into the zookeeper helper template to DRY it up throughout this chart. What do you think?

Reply:
Good idea @benjigoldberg . Please let me know if the latest commit is agreeable.